src/config.rs

```diff
@@ -23,32 +22,63 @@ const DEFAULT_SYMLINK_ROOT: &'static str = "/var/run/gpio";
 #[derive(Debug, PartialEq, Clone)]
 pub struct Direction(pub sysfs_gpio::Direction);
 
+#[derive(Deserialize, Debug)]
+#[serde(remote = "sysfs_gpio::Direction")]
+pub enum DirectionDef {
+    #[serde(rename = "in")]
+    In,
+    #[serde(rename = "out")]
+    Out,
+    #[serde(rename = "high")]
+    High,
+    #[serde(rename = "low")]
+    Low,
+}
+
 impl From<sysfs_gpio::Direction> for Direction {
     fn from(e: sysfs_gpio::Direction) -> Self {
         Direction(e)
     }
 }
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Deserialize)]
 pub struct PinConfig {
     pub num: u64,
+    #[serde(default = "default_direction")]
```

Comment:
Rather than having the function, can we just impl Default for the `Direction` newtype and then use `#[serde(default)]`